src/main/java/com/box/sdk/BoxSearchParameters.java

```diff
@@ -288,15 +291,11 @@ public void setTrashContent(String trashContent) {
         this.trashContent = trashContent;
     }
     /**
-     * Retrieve the existing BoxMetaDataFilterRequest.
-     * @return this.BoxMetaDataFilterRequest
+     * Retrieve the existing BoxMetaDataFilter.
+     * @return this.BoxMetaDataFilter
      */
-    public BoxMetadataFilterRequest getMdFilters() {
-        BoxMetadataFilterRequest bmfr = new BoxMetadataFilterRequest();
-        for (BoxMetadataFilter bmf : this.mdFilters) {
-            bmfr.addBoxMetadataFilter(bmf);
-        }
-        return bmfr;
+    public List<BoxMetadataFilter> getMdFilters() {
```

Comment:
I think we should rename this to `getMetadataFilters()` to make the name clearer.